README.md

```diff
@@ -7,24 +7,24 @@
 
```

Comment:
while we are here.. what about mentioning [the medium blog post](https://medium.com/virtuslab/take-a-look-at-your-repository-from-a-new-perspective-with-git-machete-plugin-e2e5e4197a0b) under the *References* section?

Reply:
Done